src/Diagnostics/dump_tendencies.jl

```diff
@@ -0,0 +1,233 @@
+"""
+    setup_dump_tendencies_diagnostics(
+        ::ClimateMachineConfigType,
+        interval::String,
+        out_prefix::String;
+        writer = NetCDFWriter(),
+        interpol = nothing,
+    )
+
+Create the "DumpTendencies" `DiagnosticsGroup` which contains all the
+tendencies for the simulation's `BalanceLaw`. These are collected on
+an interpolated grid (`interpol` _must_ be specified).
+
+!!! warning This diagnostics group can produce a lot of output and is
+expensive.
```

Comment:
```suggestion
!!! warn
    This diagnostics group can produce a lot of output and is
    very expensive.

!!! warn
    These diagnostics are intended for physics-debugging, and not numerics debugging, because
    `flux` and `source` are called on interpolated data, which may not exactly match the
    non-interpolated evaluation of fluxes and sources that the solver uses.
```